iOS/WAStickersThirdParty/StickerPack.swift

```diff
@@ -15,12 +15,16 @@ enum StickerPackError: Error {
     case fileNotFound
     case emptyString
     case unsupportedImageFormat(String)
-    case imageTooBig(Int64)
+    case imageTooBig(Int64, Bool)
```

Comment:
Maybe it would help here to add a comment saying that the Bool is for whether or not the image is animated. It's not super obvious as it.